main/monmon.rb

```diff
@@ -1,22 +1,22 @@
 require 'csv'
 
 balance = Hash.new(0)
+not_sup = Hash.new(0)
```

Comment:
please, give full names to variables, it should be `not_supported`
when I read `not_sup`I have no idea what is sup, see what it may mean https://www.urbandictionary.com/define.php?term=sup